server/api/Pipfile

```diff
@@ -4,58 +4,26 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
+pytest-asyncio = "*"
+pytest = "*"
+flake8 = "*"
 
 [packages]
-aiofiles = "==0.4.0"
-attrs = "==19.3.0"
-certifi = "==2019.11.28"
-chardet = "==3.0.4"
-entrypoints = "==0.3"
-flake8 = "==3.7.9"
-h11 = "==0.8.1"
-h2 = "==3.1.1"
-hpack = "==3.0.0"
-httpcore = "==0.3.0"
-httptools = "==0.0.13"
-hyperframe = "==5.2.0"
-idna = "==2.9"
-importlib-metadata = "==1.4.0"
-mccabe = "==0.6.1"
-more-itertools = "==8.1.0"
-multidict = "==4.5.2"
-numpy = "==1.18.1"
-pandas = "==1.0.1"
-pluggy = "==0.13.1"
-psycopg2-binary = "==2.8.4"
-py = "==1.8.1"
-pycodestyle = "==2.5.0"
-pyflakes = "==2.1.1"
-pyparsing = "==2.4.6"
-pysupercluster = "==0.7.6"
-pytest = "==5.3.3"
-python-dateutil = "==2.8.1"
-python-dotenv = "==0.13.0"
-python-http-client = "==3.2.7"
-pytz = "==2019.3"
-redis = "==3.5.0"
-requests = "==2.23.0"
-requests-async = "==0.5.0"
-rfc3986 = "==1.3.2"
-sanic = "==19.9.0"
-sendgrid = "==6.3.1"
-six = "==1.14.0"
-sodapy = "==2.0.0"
-tabulate = "==0.8.7"
-ujson = "==1.35"
-urllib3 = "==1.25.8"
-uvloop = "==0.14.0"
-wcwidth = "==0.1.8"
-websockets = "==8.1"
-zipp = "==1.0.0"
-sanic_compress = "==0.1.1"
-Sanic-Cors = "==0.10.0.post3"
-Sanic-Plugins-Framework = "==0.9.2"
-SQLAlchemy = "==1.3.13"
+numpy = "*"
+pandas = "*"
+psycopg2-binary = "*"
+pysupercluster = "*"
+python-dotenv = "*"
+redis = "*"
+requests-async = "*"
+sanic = "*"
+sanic-plugins-framework = "*"
+sanic-cors = "*"
+sanic-compress = "*"
+sendgrid = "*"
+sodapy = "*"
+SQLAlchemy = "*"
+tabulate = "*"
```

Comment:
So if we're using a Pipfile, can we get rid of requirements.txt? We'd have to update`/server/api/Dockerfile`, but that shouldn't be a big deal.

Reply:
we still need requirements.txt. it pins specific versions of each library and its dependencies. when we do new releases we need to do a pip freeze and make sure we're testing with the right versions in each environment.

this approach will help us stay current with new libraries while also being intentional about what versions we use.

Reply:
ok sounds good